pkg/networkmanager/interfaces.js

```diff
@@ -733,8 +733,12 @@ export function NetworkManagerModel() {
                     }
                 };
             }));
-        } else {
+        } else
             delete result.wireguard;
+
+        if (settings.vpn) {
+            set("vpn", "service-type", "s", settings.vpn['service-type']);
+            set("vpn", "data", "a{ss}", settings.vpn.data);
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19363-20230921-075022-79cb9fd2-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)